### PR TITLE
Handle no IPv4 address when guessing initial DHCP

### PR DESCRIPTION
**Pull request by Mcat12**
Changes proposed in this pull request:

- Set default DHCP values to 0.0.0.0 if there's no IPv4 address

I don't know if we want to completely disable the DHCP stuff from being enabled if there's no IPv4 address. Discuss.

@pi-hole/dashboard

### CODE REVIEW

**Comment by DL6ER** [comment]
Have to tried saving that? Will is identify `0.0.0.0` as invalid IPs?

**Comment by Mcat12** [comment]
Nope, completely untested. Hopefully the user fills in the information.

File: settings.php [hunk 1]
```diff
@@ -106,10 +106,17 @@
 	{
 		$DHCP = false;
 		// Try to guess initial settings
-		$DHCPdomain = explode(".",$piHoleIPv4);
-		$DHCPstart  = $DHCPdomain[0].".".$DHCPdomain[1].".".$DHCPdomain[2].".201";
-		$DHCPend    = $DHCPdomain[0].".".$DHCPdomain[1].".".$DHCPdomain[2].".251";
-		$DHCProuter = $DHCPdomain[0].".".$DHCPdomain[1].".".$DHCPdomain[2].".1";
+		if($piHOleIPv4 !== "unknown") {
```
Comment (DL6ER) [inline settings.php]: Typo, this won't do anything

**Comment by DL6ER** [comment]
Approved

**Pull request merged**